liferay-echo/liferay-portal
# LPS-174471 + LPS-174465 Site template friendly URL conflict warnings on page trees

Dear @liferay-echo ,

Just like in a related PR https://github.com/liferay-echo/liferay-portal/pull/11948 , these two stories also modify the same code, therefore, I'm sending [LPS-174471](https://issues.liferay.com/browse/LPS-174471) and [LPS-174465](https://issues.liferay.com/browse/LPS-174465) together.

### Motivation

The aim is to display more user friendly warnings about friendly URL conflicts between the pages of site templates and the sites created from them. These stories add warnings on the page trees in 2 places:
* Site Builder > Pages
* Home > Page Tree

### Testing steps

1. Create a new site template: 'temp1'
2. Create a new site from the template: 'site1'
3. Add a new page to 'site1', name it "ConflictPage"
4. Add a new page to 'temp1', also named "ConflictPage"

* Both of the friendly URLs will be "/conflictpage", therefore, they will be colliding.
* Go to "Site Builder > Pages", verify that a "!" warning sign appears next to the page name. Tooltip on the warning shows more detailed info.
* Go to "Home > Page Tree", verify that a "!" warning sign appears next to the page name. Tooltip on the page name shows more detailed info.
* These warnings should work from the template's ([LPS-174465](https://issues.liferay.com/browse/LPS-174465)) and the site's ([LPS-174471](https://issues.liferay.com/browse/LPS-174471)) side as well, just with different tooltip messages.

### Feature flag
```
feature.flag.LPS-174471=true
```

### Technical tasks

Assigned [LPS-183879](https://issues.liferay.com/browse/LPS-183879) and [LPS-183880](https://issues.liferay.com/browse/LPS-183880) technical tasks to keep track of the reviews.

Thanks,
Vendel
